src/utils/configLoad.ts

```diff
@@ -37,23 +37,20 @@ const maybeInjectAnalyticsLibrary = (config: Config) => {
 };
 
 const calculateAccessModel = (config: Config): AccessModel => {
-  if (config?.integrations?.cleeng) {
-    const { id, monthlyOffer, yearlyOffer } = config?.integrations?.cleeng || {};
+  if (config?.integrations?.cleeng?.id) {
+    if (config?.integrations?.cleeng?.monthlyOffer || config?.integrations?.cleeng?.yearlyOffer) {
```

Comment:
Probably ternary operator could simplify it for cleeng and inplayer.

```
const {cleeng, jwp} = config?.integrations || {};

if (cleeng?.id) {
    return cleeng?.monthlyOffer || cleeng?.yearlyOffer ? 'SVOD' : 'AUTHVOD';
}

if (jwp?.clientId)  {
    return jwp?.assetId ? 'SVOD' : 'AUTHVOD';
}

return 'AVOD';

```

Reply:
Nice, updated